cli/ioctl/cmd/account/accountdelete.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"syscall"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"golang.org/x/crypto/ssh/terminal"
+	"gopkg.in/yaml.v2"
+
+	"github.com/iotexproject/iotex-core/address"
+	"github.com/iotexproject/iotex-core/cli/ioctl/cmd/config"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// accountDeleteCmd represents the account delete command
+var accountDeleteCmd = &cobra.Command{
+	Use:   "delete name/address",
+	Short: "delete an IoTeX keypair from wallet",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(accountDelete(args))
+	},
+}
+
+func accountDelete(args []string) string {
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err.Error()
+	}
+	name, addr := "", ""
+	for nameInList, addrInList := range cfg.AccountList {
```

Comment:
for name, addr = range cfg.AccountList {

then don't need nameInList, addrInList